dist/stat_data.py

```diff
@@ -545,7 +545,7 @@ def __init__(self, name, desc, flags=''):
     RecStat('rec_time_aggr_newest_stop_ts', 'pages written including an aggregated newest stop timestamp '),
     RecStat('rec_time_aggr_newest_stop_txn', 'pages written including an aggregated newest stop transaction ID'),
     RecStat('rec_time_aggr_oldest_start_ts', 'pages written including an aggregated oldest start timestamp '),
-    RecStat('rec_time_aggr_oldest_start_txn', 'pages written including an aggregated oldest start transaction ID '),
+    RecStat('rec_time_aggr_newest_txn', 'pages written including an aggregated newest transaction ID '),
```

Comment:
I think for consistency this should be newest_start_txn, given that we use newest_start_ts and newest_start_durable_ts?

Reply:
The `newest_txn` stores the maximum transaction id that is present as part of both `start transaction` and the `stop transaction` due to the aggregated `newest stop transaction` can include the `max transaction` also. Due to this reason, I didn't include the `start` as part of the name.

Reply:
As in I can commit a deletion which will then be the newest txn on the page?

Reply:
Yes. your understanding is correct.